apps.yaml

```diff
@@ -43,7 +43,10 @@ nanoribbons:
         title: Empa nanotech@surfaces Laboratory - Graphene nanoribbons
 quantum-espresso:
   releases:
-    - url: 'git+https://github.com/aiidalab/aiidalab-qe.git@master:v20.11.0^..'
+    - url: 'git+https://github.com/aiidalab/aiidalab-qe.git@master:v22.05.0^..'
+      metadata:
```

Comment:
If you provide metadata under an URL - it should be complete, I believe. You can't provide just the `state` key, but all the keys. To my understanding, it should be either only URL or URL + **complete** metadata.

@csadorf correct me if I am wrong

Reply:
Yes, that's correct. If you provide metadata for release line, it needs to be complete, it is not merged.